CHANGELOG.rst

```diff
@@ -14,6 +14,36 @@ Unreleased
 
 *
 
+[2.0.0] - 2025-01-23
+**********************************************
+
+Added
+=====
+
+* Support for Python 3.11 and Django 4.2
+* Add  github workflow
+* Remove support to "redirect to another unit in the same subsection" action
+* Fix applyFlowControl in injection to applied correctly the actions of flow-control 
```

Comment:
This should be in the [changed](https://github.com/eduNEXT/flow-control-xblock/pull/54/files#diff-2c623f3c6a917be56c59d43279244996836262cb1e12d9d0786c9c49eef6b43cR34) section